### PR TITLE
Improved error message, code refactoring

### PR DESCRIPTION
**Pull request by kkrampa**
@kaapstorm @czue 
http://manage.dimagi.com/default.asp?228543

### CODE REVIEW

File: corehq/apps/importer/views.py [hunk 1]
```diff
@@ -298,19 +297,17 @@ def importer_job_poll(request, domain, download_id, template="importer/partials/
...
-        # todo: and HttpResponseRedirect
         error = e.errors
         if error == 'EXPIRED':
-            return _spreadsheet_expired(request, domain)
+            error = _('Sorry, your session has expired. Please start over and try again.')
         elif error == 'HAS_ERRORS':
-            messages.error(request, _('The session containing the file you '
-                                      'uploaded has expired - please upload '
-                                      'a new one.'))
+            error = _('The session containing the file you uploaded has expired - please upload a new one.')
```
Comment (kaapstorm) [inline corehq/apps/importer/views.py]: It seems like this error message might not be accurate (although I appreciate you're just keeping the original error message). `HAS_ERRORS` occurs on `ImporterError`, which could be raised on file not found, or if there is a problem with the spreadsheet itself. If we are checking explicitly for `ImporterRefError`, we might want to check for `ImporterFileNotFound` and `ImporterExcelError` separately too. Just a suggestion though. Not blocking.

Comment (kkrampa) [inline corehq/apps/importer/views.py]: I will change this.

**Comment by kkrampa** [comment]
@kaapstorm Updated

**Comment by kaapstorm** [comment]
Thanks for adding that @kkrampa. Looks good to me.

**Pull request merged**